adr/0017-hardware-screening-os.md

```diff
@@ -0,0 +1,34 @@
+# 0017. Hardware screening for OS
+
+Date: 2020-10-23
+
+## Status
+
+Draft
+
+## Context
+
+We need take a decision for the minimal requirements of a IoT-Board to become a Home Assistant OS image.
+Also how we deal with old or replaced generation/boards.
+
+## Decision
+
+Follow are the requirements of a Board:
+
+- The board needs to be available for private customers in most major economic regions
+- The manufacturer produces and sells the boards for at least 3 years
```

Comment:
This will leave out RPi 4 and ODROID N2, those are not even 3y old.

Reply:
I assume they mean that it will be produced for 3 years? So not a "special occasion board for the anniversary of something"-board.

Basically, most boards are considered outdated after 3 years...

Reply:
What bord manufacture advertises how long a board will be produced?

Reply:
Not many I guess, maybe @pvizeli or @balloob can explain what is meant with this requirement.

Reply:
Correct, there are many SBC manufacture which only sells a board 6-1y. That means, after we added the support, they are no longer available for new users. Special OrangePi is manufacture like this. Make all 3-6 months a new board. Since we need to buy new hardware for tests and don't make all the effort for something which is not available. RaspberryPi, ODROID is a manufacture which can be used also on B2B because these boards are a long time available and supported-